apps/src/StudioApp.js

```diff
@@ -3039,25 +3065,49 @@ function rectFromElementBoundingBox(element) {
  * @param {string} type - Alert type (error, warning, or notification)
  * @param {React.Component} alertContents
  */
-StudioApp.prototype.displayWorkspaceAlert = function(type, alertContents) {
-  var container = this.displayAlert(
-    '#codeWorkspace',
-    {type: type},
-    alertContents
-  );
+StudioApp.prototype.displayWorkspaceAlert = function(
```

Comment:
Can you refactor this out into its own react component?

Reply:
Context: we probably should have set this up originally as its own component, but now that we're adding more logic and functionality, it would be appropriate to do that now.

Reply:
We dug into this and decided against it for now.